### ISSUE TITLE
Queued playlist only play few items then stoped. hardly play to end.

### ISSUE THREAD

**Issue by newbdez33** [dimitris-c/AudioStreaming#11]
It is almost 100% reproduced if there are many queued mp3 urls in queue.
I created a repo for this issue.

https://github.com/newbdez33/AudioStreamingBug

The basic code is queue all playlist items to player.queue()
```swift
for item in playlist {
            if let url = URL(string: item) {
                print("queue:\(url)")
                player.queue(url: url)
            }
}
```

Sometimes it plays 2 or 3 items then stoped.
Tested main branch in iOS 14.2 Simulator and iPhone 7. (Xcode 12.2 and Xcode 12.3)

**Comment by dimitris-c** [comment]
Yeap this is happening indeed, I realized it last week and started working on a fix...

**Comment by dimitris-c** [comment]
Fixed in #13.
